File: DEBERES/Deberes en js y ts/04-SWITCHCondicional.ts
/*
Este programa define si la division de dos numeros es exacta o no.
*/
console.log("\tBienvenido, este programa define si la division entre dos numeros es exacta.\n\n");
const primerNumString = prompt("Ingrese el primer numero: ");
const segundoNumString = prompt("Ingrese el segundo numero: ");
const numUno = Number(primerNumString);
const numDos = Number(segundoNumString);

const resultado = (numUno/numDos);
const residuo = numUno%numDos;

if(numDos != 0){
    
    switch(residuo){
    case 0: 
        console.log("El resultado de la division es: " + resultado + "\nPor lo tanto la division es EXACTA. ");
        break;
    default:
        console.log("El resultado de la division es: " + resultado + "\nPor lo tanto la division NO es EXACTA. ");
        break;
    }

}else{
    alert("No se puede dividir para 0.");
}

File: DEBERES/Deberes en mi lenguaje C++/2019/04-SWITCHCondicional.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    /*
Este programa define si la division de dos numeros es exacta o no.
*/
cout<<"\tBienvenido, este programa define si la division entre dos numeros enteros es exacta.\n\n";
int numUno, numDos, resultado;
cout<<"Ingrese el primer numero: \n"; cin>>numUno;
cout<<"Ingrese el segundo numero: \n"; cin>>numDos;


resultado = (numUno/numDos);
int residuo = numUno % numDos;


if(numDos!= 0){
    
    switch(residuo){
    case 0: 
        cout<<"El resultado de la division es: "<<resultado<<"\nPor lo tanto la division es EXACTA. ";
        break;
    default:
        cout<<"El resultado de la division es: "<<resultado<<"\nPor lo tanto la division NO es EXACTA. ";
        break;
    }

}else{
    cout<<"No se puede dividir para 0.";
}




    return 0;
}